frontend/src/components/NestedAccordion.tsx

```diff
@@ -20,7 +20,15 @@ const NestedAccordion: React.FC<NestedAccordionProps> = ({
   tasks = [],
 }) => {
   return (
-    <Accordion className="mb-4">
+    <Accordion className="mb-4"
+    sx={{
+      "@media print": {
+        ".mui-style-bz4dnt-MuiCollapse-root": {
```

Comment:
This goes back to one of Greg's comments on another PR.  It should be possible to exactly target this element without the dynamic [bs4dnt]...albeit kind of annoying/trial and error.

Reply:
Fixed!